template/js/netlify-cms/preview/home-preview.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+export default class HomePreview extends React.Component {
+  constructor () {
+    super()
+    this.state = {
+      doc: '',
+      html: ''
+    }
+  }
+
+  componentDidMount () {
+    this.fetchPage()
+  }
+
+  async fetchPage () {
+    fetch('/index.html').then(response => {
+      return response.text()
+    })
+      .then(html => {
+        // Initialize the DOM parser
+        const parser = new DOMParser()
+        const doc = parser.parseFromString(html, 'text/html')
+        this.setState({ html, doc })
+      })
+      .catch(err => {
+        console.log('Failed to fetch page: ', err)
+      })
+  }
+
+  render () {
+    const { doc } = this.state
+    let newDoc = doc
+    let html
+    if (doc) {
+      const { entry } = this.props
+      const data = JSON.parse(entry.getIn(['raw']))
+
+      // set slider
+      this.props.widgetsFor('slider').getIn(['data', 'slides']).map(function(slider, index) {
+        // set last img has preview for slider
+        const img = slider.getIn(['img'])
+        const $slider = doc.querySelectorAll('.glide')
+        if (img) {
+          $slider[0].classList.add(...['glide--ltr', 'glide--slider', 'glide--swipeable'])
+          const $imgs = doc.querySelectorAll('.banner-slider img')
+          for (let i = 0; i < $imgs.length; i++) {
+            const $img = $imgs[i]
+            $img.classList.remove()
+            $img.classList.add(...['lozad', 'fade', 'img-fluid', 'show'])
+            $img.setAttribute('src', img)
+            $img.setAttribute('data-loaded', true)
+          }
+          $slider[0].style.display = 'block'
+        } else {
+          $slider[0].style.display = 'none'
+        }
+      })
```

Comment:
Eu acho que não precisaria disso aqui não.

E se você só subscrevesse o CSS padrão do glide com alguns outros estilos no `previews.scss`?
Com estes novos estilos você mostraria apenas a primeira imagem, as outras simplesmente seriam ignoradas a princípio, e então você poderia usar `data-cms-set="src:home.slider[0]"` (só um exemplo, não olhei se _home.slider[0]_ já é o src da imagem) no elemento da primeira imagem, o que acha?

No melhor cenário você poderia aplicar o data `data-cms-set="src:home.slider[i]"` em todas as imagens e de fato fazer o slider funcionar importando o `./lib/glide-slides` (não sei se funcionaria out of the box, se for complicado podemos ir com a outra opção por enquanto).